core/App/screens/ProofRequest.tsx

```diff
@@ -137,10 +137,10 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
   }, [])
 
   useEffect(() => {
-    if (proof.state === ProofState.Done) {
-      pendingModalVisible && setPendingModalVisible(false)
-      setSuccessModalVisible(true)
-    }
+    // if (proof.state === ProofState.Done) {
```

Comment:
Generally looks good, probably just remove commented code that's not required.

Reply:
Fixed.